### DISCUSSION TITLE
Play Store beta launch?

### DISCUSSION THREAD

**Discussion by R0X-IT** [skyline-emu/skyline#310] [General]
Hello friends.  Any predictions when the beta will be released on the Playstore?  There could be an option inside the app to donate (like in citra), unlocking some bullshit like dark mode, that would help the team too.  The emulator is getting fantastic, congratulations!

**Comment by iniMerchant64** [comment]
Not anytime soon thats for sure. Developers plan on supporting a bigger library of games before skyline hits the play store but itll do happen eventually. Also as far as i can tell there wont be a premium version or stuff unlockable by donations but there will be an option to donate and possible optional ads in the future
